bin/docker_push

```diff
@@ -1,43 +1,25 @@
 #!/bin/bash -e
 
-# Travis CI seems to source this script, so it is ignoring the #! above!
 set -e
 
-if [ -n "$DAQ_PUSH" ]; then
-    VERSION=$(cat misc/RELEASE_VERSION)
-    TARGET=$DAQ_PUSH:$VERSION
-    RTAG=release-$VERSION
-
-    echo Doing push to target $TARGET
-
-    head_hash=$(git rev-list -n 1 HEAD)
-    version_hash=$(git rev-list -n 1 $RTAG)
-
-    if [ $head_hash != $version_hash ]; then
-	echo Release head does not match version tag.
-	echo head=$head_hash
-	echo rtag=$version_hash
-	false
-    fi
-		   
+if [ -n "$TARGET" ] && [ -n "$VERSION" ]; then
```

Comment:
TARGET and VERSION seem a bit too generic for this behavior -- they could very easily overlap with something else and cause problems -- prefix with DAQ_ to provide a namespace for them...